docker-images/artifacts/kafka-thirdparty-libs/4.0.x/pom.xml

```diff
@@ -278,6 +278,30 @@
                     <groupId>org.apache.kafka</groupId>
                     <artifactId>kafka-clients</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.apache.kafka</groupId>
+                    <artifactId>kafka-server</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.kafka</groupId>
+                    <artifactId>kafka-server-common</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.kafka</groupId>
+                    <artifactId>kafka-metadata</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.kafka</groupId>
+                    <artifactId>kafka-raft</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.kafka</groupId>
+                    <artifactId>kafka-group-coordinator</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.kafka</groupId>
+                    <artifactId>kafka-storage</artifactId>
+                </exclusion>
```

Comment:
How did you come up with these exclusions? They do not seem to be a dependcies in the first place, so why exclude them?

Reply:
When we build the operator and push the images, I was getting a lot collisions w.r.t these dependencies

Reply:
I can see some of the dependencies added in the `build.gradle` file -> 
```
    api "org.apache.kafka:kafka-server:$kafkaVersion"
    api "org.apache.kafka:kafka-server-common:$kafkaVersion"
    api "org.apache.kafka:kafka-storage:$kafkaVersion"

````

Reply:
You should likely check the pom file(s) and not the Gradle build files which mean many things. Also, if you see only "some" of them, it confirms that this is wrong and should be fixed.

Reply:
Thanks for the review Jakub, I was able to remove extra exclusions since I saw `kafka-server` is the only exclusion which was required. Itt was the one bringing in all the other stuff. Thanks again